Documentation/Articles/Customise-loading-of-resources.md

```diff
@@ -0,0 +1,59 @@
+# Customise loading of resources
```

Comment:
Don't forget the entry in the TOC (`Documentation/README.md`). In fact, we should write a Danger rule for that.